tests/behat/features/bootstrap/TopicContext.php

```diff
@@ -124,6 +124,29 @@ public function createTopics(TableNode $topicsTable) : void {
     }
   }
 
+  /**
+   * Create multiple topics at the start of a test with defined users as
+   * authors.
+   *
+   * Creates topics provided in the form:
+   * | title    | author    | field_topic_type | body           | field_content_visibility | langcode |
+   * | My Topic | My author | News             | My description | public                   | en       |
+   * | ...      | ...       | ...              | ...            | ...                      | ...      |
+   *
+   *
+   * @Given topics with defined author:
+   */
+  public function createTopicsWithDefinedAuthor(TableNode $topicsTable) : void {
```

Comment:
@volodymyr-sydor I think we can use existing method `createTopics` for it?

Reply:
@ribel - the same here, provided users should be used as authors for the creation of content